client/state/selectors/get-active-discount.js

```diff
@@ -56,9 +65,7 @@ export default state => {
 
```

Comment:
It's unrelated to this PR, but `activeDiscounts.filter` above should be replaced with `activeDiscounts.find`.

Reply:
Yup, nice small fix. Thanks!